src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -43,7 +43,7 @@ websocket.manual_send.fail.invalid_opcode       = Invalid WebSocket opcode selec
 websocket.manual_send.menu                      = Manual Send WebSocket Message
 websocket.manual_send.menu.mnemonic				= w
 websocket.manual_send.popup                     = Resend WebSocket Message
-websocket.manual_send.resend.menu               = Resend...
+websocket.manual_send.resend.menu               = Open/Resend with Request Editor...
```

Comment:
So if we use `WebSocket Message Editor` for the titles then this would become `Open/Resend with Message Editor...`